grootfs.c

```diff
@@ -948,7 +949,7 @@ grootfs_setxattr (const char *path, const char *name, const char *value,
     return dirfd;
 
   autofree char *proc_file = get_proc_fd_path (dirfd, basename);
-  autofree char *fake_name = xasprintf ("user.grootfs.%s", name);
```

Comment:
There is no need to dynamically compose the two, you should be able to (with ATTR_PREFIX being a define) to do `xasprintf(ATTR_PREFIX".%s", name);